src/Validation.js

```diff
@@ -0,0 +1,32 @@
+const ERROR = {
+  not_number: "[ERROR] 숫자를 입력해야 합니다.",
+  out_of_range: "[ERROR] 다리 길이는 3부터 20 사이의 숫자여야 합니다.",
+  not_u_or_d: '[ERROR] 이동할 칸은 U나 D만 입력할 수 있습니다.',
+  not_uppercase: '[ERROR] 대문자로 입력해주세요.',
+  not_r_or_q: '[ERROR] R이나 Q만 입력할 수 있습니다.'
+};
+
+const checkBridgeSize = (size) => {
+  if (/[^\d]+/g.test(size)) throw new Error(ERROR.not_number);
+  if (size < 3 || size > 20) throw new Error(ERROR.out_of_range);
+  return size;
+};
+
+const checkMovingSpace = (space) => {
+  if (space === 'u' || space === 'd') 
```

Comment:
```suggestion
  if (space.toLowerCase()) 
```
toLowerCase()를 이용해서 소문자 검사를 해도 좋을 것 같아요